content/news/2025-01-22_rocur-relaunch/index.en.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "We're Launching the R-Ladies Rotating Curation (RoCur) Account on Bluesky!"
+author: "RoCur team"
+type: blog
+date: "2025-01-22"
+image: img/bluesky_media_kit_logo_1.png
+tags: 
+- rocur
+categories: 
+- R-Ladies
+summary: We're looking for curators for the We Are R-Ladies Bluesky account!
+slug: rocur-relaunch-2025
+---
+
+![Bluesky logo. A white butterfly on a blue gradient.](img/bluesky_media_kit_logo_1.png)
+
+**Caption:** Bluesky logo, sourced from the official [Bluesky media kit](https://drive.google.com/drive/folders/16mlJPfWNnc6jj-3vGZ88SFysIY13GBp0).
+
+We're absolutely thrilled to announce that the [R-Ladies Rotating Curation (RoCur) account](https://bsky.app/profile/weare.rladies.org) is now live on Bluesky! Each week, a new member of the R-Ladies community will take over the account to talk about all things R. This will be a fantastic opportunity for individuals who identify as a minority gender in the R community to amplify their voices and connect with others in the greater data science community.
```

Comment:
```suggestion
We are absolutely thrilled to announce the R-Ladies Rotating Curation (RoCur) Bluesky account [bsky.app/profile/weare.rladies.org](https://bsky.app/profile/weare.rladies.org) is now live! Each week, a new member of the R-Ladies community will take over the account to talk about all things R. This will be a fantastic opportunity for individuals who identify as a minority gender in the R community to amplify their voices and connect with others in the greater data science community.
```